### ISSUE TITLE
Source code edit? Insert <script> tag

### ISSUE THREAD

**Issue by cimenta** [GetmeUK/ContentTools#585]
Hi

I am looking for an editor for a blog site and time to time the users need to insert  and later on also edit <script> tag into the post. But in a way that the script actually works.

Does this editor allow users to insert and save working <script> tag?

Thank you
Radek

**Comment by anthonyjb** [comment]
You can insert/modify raw HTML, including script tags by clicking on a HTML tag in the editors tags path (e.g P or UL > LI) (bottom left), then selecting the source tab in the window that appears.

However, out of the box that's about it there not a first class element type you can edit with CT.

**Comment by cimenta** [comment]
sorry I did not understand.. So it it possible or not? 
Actually I want to insert and then edit the <script> tag

**Comment by anthonyjb** [comment]
Yes - it is possible since you can edit the underlying HTML via the editor - you'll likely find this is the case with most editors.
